### PR TITLE
Add callable_mp_static

### PR DESCRIPTION
**Pull request by reduz**
Allows using a static (C) function as a callable.

### CODE REVIEW

File: core/object/callable_method_pointer.h [hunk 1]
```diff
@@ -245,4 +245,87 @@ Callable create_custom_callable_function_pointer(T *p_instance,
...
+	} data;
+
+public:
+	virtual ObjectID get_object() const {
+		return ObjectID();
+	}
+
+	virtual void call(const Variant **p_arguments, int p_argcount, Variant &r_return_value, Callable::CallError &r_call_error) const {
+		call_with_variant_args_static_ret(data.method, p_arguments, p_argcount, r_return_value, r_call_error);
+		r_return_value = Variant();
```
Comment (BastiaanOlij) [inline core/object/callable_method_pointer.h:263]: Is this just to clear any return value?

File: core/object/callable_method_pointer.h [hunk 2]
```diff
@@ -245,4 +245,87 @@ Callable create_custom_callable_function_pointer(T *p_instance,
...
```
Comment (BastiaanOlij) [inline core/object/callable_method_pointer.h:301]: in this case, shouldn't be doing this right? as we want the return value? or am I reading the code wrong?

Comment (reduz) [reply core/object/callable_method_pointer.h]: thanks for spotting it!

**Review by BastiaanOlij — APPROVED** [review]
Looks good to go to me !

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**